lmfdb/modular_curves/family.py

```diff
@@ -1,36 +1,45 @@
-
 from sage.misc.lazy_attribute import lazy_attribute
 from lmfdb.modular_curves.web_curve import get_bread
-from lmfdb.utils import display_knowl
+
 
 def ModCurveFamily(name):
-    if name == "X0":
-        return X0N()
-    elif name == "X1":
-        return X1N()
-    elif name == "X":
-        return XN()
-    elif name == "Xpm1":
-        return Xpm1N()
-    elif name == "Xarith1":
-        return Xarith1MMN()
-    elif name == "Xarithpm1":
-        return Xarithpm1MMN()
-    elif name == "Xsp":
-        return XspN()
-    elif name == "Xspplus":
-        return XspplusN()
-    elif name == "Xns":
-        return XnsN()
-    elif name == "Xnsplus":
-        return XnsplusN()
-    elif name == "XS4":
-        return XS4p()
-    elif name == "Xarith":
-        return XarithN()
-    raise ValueError("Invalid name")
+    return get_family(name)
```

Comment:
You should have a function that just directly calls another function.  I think you want to replace the contents of `def ModCurveFamily(name):` with what you write below, and just not have a function `get_family`.